tests/AuditableBehaviorTest.php

```diff
@@ -77,4 +76,68 @@ public function testMappedOriginalValues()
         $this->assertEquals($testObject->getOriginalFieldValues(true), array('id' => 101, 'email_address' => 'reynaldocastellano@gmail.com'));
     }
     
+    public function testWasNewPropertyDeclaration()
+    {
+    	$testObject = new TestModel();
+        $refTestObject = new ReflectionObject($testObject);
+        $this->assertTrue($refTestObject->hasProperty('wasNew'), 'The property "was_new" was NOT declared in model');
+    }
+
+    public function testWasModifiedPropertyDeclaration()
+    {
+    	$testObject = new TestModel();
+        $refTestObject = new ReflectionObject($testObject);
+        $this->assertTrue($refTestObject->hasProperty('wasModified'), 'The property "was_modified" was NOT declared in model');
+    }
+
+    public function testWasNewMethodDeclaration()
+    {
+        $testObject = new TestModel();
+        $refTestObject = new ReflectionObject($testObject);
+        $this->assertTrue($refTestObject->hasMethod('getWasNew'));
+    }
+
+    public function testWasModifiedMethodDeclaration()
+    {
+        $testObject = new TestModel();
+        $refTestObject = new ReflectionObject($testObject);
+        $this->assertTrue($refTestObject->hasMethod('getWasModified'));
+    }
+
+    public function testPreservationOfWasNewAfterSave()
+    {
+        $origValues = array(101, 'reynaldocastellano@gmail.com');
+        $testObject = new TestModel();
+        // $testObject->setId(101);
+        // $testObject->setEmailAddress('logancastellano@gmail.com');
+        // $this->assertEquals(false, $testObject->getWasNew());
+        // $testObject->save();
+        // $this->assertEquals(true, $testObject->getWasNew());
+        // $testObject->setEmailAddress('reynaldocastellano@gmail.com');
+        // $this->assertEquals(true, $testObject->getWasNew());
+        // $testObject->save();
+        $this->assertEquals(false, $testObject->getWasNew());
```

Comment:
Just a typo. It was supposed to be true since the setters were commented out. My bad.